sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateLibraryTemplateDao.java

```diff
@@ -0,0 +1,124 @@
+package uk.ac.bbsrc.tgac.miso.persistence.impl;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.criterion.Restrictions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import uk.ac.bbsrc.tgac.miso.core.data.impl.LibraryTemplate;
+import uk.ac.bbsrc.tgac.miso.core.store.LibraryTemplateStore;
+import uk.ac.bbsrc.tgac.miso.core.util.DateType;
+
+@Repository
+@Transactional(rollbackFor = Exception.class)
+public class HibernateLibraryTemplateDao implements LibraryTemplateStore, HibernatePaginatedDataSource<LibraryTemplate> {
+
+  protected static final Logger log = LoggerFactory.getLogger(HibernateLibraryTemplateDao.class);
+
+  private final static String[] SEARCH_PROPERTIES = new String[] {};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)